src/tests/system/lib/sssd/utils/authentication.py

```diff
@@ -3,17 +3,21 @@
 from __future__ import annotations
 
 from datetime import datetime
-from typing import Any
```

Comment:
`Any` is definitely still used. You must have replaced `Any` with `TYPE_CHECKING` by accident, instead of including `TYPE_CHECKING`.